src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/AuthenticationHelper.cs

```diff
@@ -159,6 +159,7 @@ private static void SetBasicAuthToken(HttpRequestMessage request, NetworkCredent
             // Any errors in obtaining parameter return false and we don't proceed with auth
             if (string.IsNullOrEmpty(parameter))
             {
+                if (NetEventSource.IsEnabled) NetEventSource.Error(null, "Null parameter");
```

Comment:
I think it would be better to add tracing to GetDigestTokenForCredential to indicate the specific failure.  By the time we get here, all we know is that something failed in processing the challenge.

Reply:
Fixed.